Dockerfile

```diff
@@ -36,6 +36,13 @@ RUN curl -sSLo /tmp/asset-devworkspace-operator.zip https://api.github.com/repos
     curl -sSLo /tmp/asset-devworkspace-che-operator.zip https://api.github.com/repos/che-incubator/devworkspace-che-operator/zipball/${DEV_WORKSPACE_CHE_OPERATOR_VERSION} && \
     curl -sSLo /tmp/asset-restic.zip https://api.github.com/repos/restic/restic/zipball/${RESTIC_TAG}
 
+RUN mkdir -p $GOPATH/restic && cd $GOPATH/restic && \
+    unzip /tmp/asset-restic.zip -d /tmp && \
+    mv /tmp/restic-*/* $GOPATH/restic && \
+    go mod vendor && \
```

Comment:
will need to remove this line when running in brew, but that should be an easy sed yank.